en/sql/schema/index_stmt.rst

```diff
@@ -342,8 +342,7 @@ The following is a syntax of rebuilding an index.
     *   From CUBRID 10.0, table name should not be omitted.
     
     *   From CUBRID 10.0, even if you add column names at the end of a table name, these will be ignored and recreated with the same columns with the previous index.
-
-    *   Prefix index feature is deprecated, so it is not recommended anymore.
+    
 
```

Comment:
Why remove the line ?
"Prefix index feature is deprecated, so it is not recommended anymore."

Reply:
This is mentioned in “NOTE” in the “CREATE INDEX” description.
Also, there is no need to mention the prefix index here since the ALTER INDEX syntax cannot change the configuration.

Reply:
OK, i see